covid_xprize/oxford_data/oxford_data.py

```diff
@@ -234,6 +230,11 @@ def prepare_cases_dataframe(data_url: str, threshold_min_cases=False) -> pd.Data
     return df
 
 
+def threshold_min_cases(df: pd.DataFrame) -> pd.DataFrame:
+    # Remove all rows with too few cases
```

Comment:
You could convert this comment into a proper docstring, like for the other functions in this file